db.js

```diff
@@ -43,6 +43,16 @@ var books = (function () {
       return rows[0].count;
     });
 
+    if (mostRecentLoan) {
+      var owner = await users.getById(book.user_id, currentUserId);
+      var requester = await users.getById(
```

Comment:
Is it guaranteed that the requester of this loan is the current user?

Reply:
Maybe the parameter ordering is confusing, but the signature is `users.getById(currentUserId, targetUserId)`. In short, it's like, here's who I am, now who is this person? (because the status is contextual based on who you are). 

Tldr, the user retrieved here is the `mostRecentLoan.requester_id`